dev/core/src/com/google/gwt/core/ext/soyc/impl/StoryRecorder.java

```diff
@@ -351,7 +348,9 @@ private void recordStory(SourceInfo info, int fragment, int length, Range range)
       theStory = new StoryImpl(storyCache.get(info), length);
     }
 
-    emitStory(theStory, range);
+    if (range.getStart() < js[curHighestFragment].length()) {
```

Comment:
the check `start < length` was added outside of `emitStory`, check that `end <= length` was added inside  that method -- maybe both checks should be in one place -- inside `emitStory`.

Reply:
I do not think so.
It seems that the value of the end of range property is wrong.
The call to emitStory must be kept to get the description of the story.

Reply:
OK, feel free to disregard